R/look_up.R

```diff
@@ -0,0 +1,364 @@
+#' Look up geological intervals and assign ICS ages and stages
```

Comment:
I put it in the description, I think we should reference the International Commission on Stratigraphy (ICS), rather than the Chart (I can't recall seeing "ICC" being used as an abbreviation)